v0.5/fastn-unresolved/src/debug.rs

```diff
@@ -56,14 +56,92 @@ impl fastn_jdebug::JDebug for fastn_unresolved::AliasableIdentifier {
 
 impl fastn_jdebug::JDebug for fastn_unresolved::ComponentInvocation {
     fn debug(&self) -> Value {
-        todo!()
+        serde_json::json!({
+            "content": self.name.debug(),
+            "caption": self.caption.debug(),
```

Comment:
Do these things conditionally, else test functions will become verbose. If we just had to do this then we would not need a custom trait and can use serde (in fact now that we do not depend on source for create value, we can maybe write an auto impl for everything implementing serde::Serialize so this kind of impls are not needed, and you only implement JDebug when you want to do "better than serde". 

The point of JDebug is to make the output as compact as possible so test cases look clean. The cleaner the test case, the more test cases we would write.